server/test/controllers/deployments_controller_test.rb

```diff
@@ -58,11 +58,21 @@ def setup
       assert_nil Deployment.find_by_name @deployment.name
     end
 
-    test "deploy request should successfully deploy deployment" do
-      skip # this one does not work yet, investigate why "undefined method `owner_details' for nil:NilClass"
-      response = JSON.parse(put(:deploy, :id => @deployment.id).body)
+    test "deploy endpoint should set deployment object uuid if request is successful" do
+      successful_manage_manifest_id  = '3f486234-d6e2-45a2-8403-e1920ba9cefa'
+      running_depl_task_id           = '9fd51c32-28f7-4379-b42e-362f6b2f4982'
```

Comment:
Can we just refer to the fixtures directly instead of copy pasting those ids?

```ruby
foreman_tasks_tasks(:successful_manage_manifest).id
foreman_tasks_tasks(:running_depl_task).id
```